streams/src/test/java/org/apache/kafka/streams/integration/IQv2StoreIntegrationTest.java

```diff
@@ -800,13 +914,13 @@ public void verifyStore() {
                     throw new AssertionError(queryResult.toString());
                 }
                 assertThat(partitionResult.getFailureReason(), is(FailureReason.UNKNOWN_QUERY_TYPE));
-                assertThat(partitionResult.getFailureMessage(), is(
+                assertThat(partitionResult.getFailureMessage(), matchesPattern(
                     "This store"
-                        + " (class org.apache.kafka.streams.state.internals.MeteredTimestampedWindowStore)"
+                        + " \\(class org.apache.kafka.streams.state.internals.Metered.*WindowStore\\)"
```

Comment:
Regardless of whether you supply a bytes store to the DSL, you wind up wrapped in a Timestamped store. Now that we're registering the non-timestamped store directly with the PAPI, we can also get a non-timestamped metered store, hence swapping this for a regex that matches both `MeteredWindowStore` and `MeteredTimestampedWindowStore`.